python/packages/core/agent_framework/_workflows/_group_chat.py

```diff
@@ -366,7 +366,7 @@ async def _apply_directive(
             self._conversation.extend((final_message,))
             self._history.append(_GroupChatTurn(self._manager_name, "manager", final_message))
             self._pending_agent = None
-            await ctx.yield_output(final_message)
+            await ctx.yield_output(list(self._conversation))
```

Comment:
I can't see where `self._conversation` is defined, but do we need this cast to list?

Reply:
Already a list - good catch. No need for that cast.